test/fit_tests/common/fit_common.py

```diff
@@ -587,18 +587,23 @@ def get_node_sku(nodeid):
                 if VERBOSITY >= 2:
                     errmsg = "Error: SKU API failed {}, return code {} ".format(sku, skudata['status'])
                     print errmsg
+                    return "unknown"
         else:
-            if VERBOSITY >= 2:
-                errmsg = "Error: nodeid {} did not return a valid sku in get_rackhd_nodetype{}".format(nodeid,sku)
-                print errmsg
+            return "unknown"
```

Comment:
is the string "unknown" here a better response than either an empty string or a null response? I suspect a semantically better response would be to return null rather than a value, which could be checked more easily to understand if the result wasn't a value as opposed to looking for a specific string `unknown` that represented an unknown or unset value for the SKU name